src/jira/client/axios.ts

```diff
@@ -157,6 +157,16 @@ const instrumentFailedRequest = (baseURL: string, logger: Logger) => {
 				}
 			}
 		}
+		if (error.response?.status === 405) {
+			try {
+				await axios.get("/status", { baseURL });
+			} catch (e) {
+				if (e.response.status === 302) {
+					logger.info(`405 from Jira: Jira instance '${baseURL}' has been renamed. Returning 404 to our application.`);
+					error.response.status = 404;
+				}
+			}
+		}
 		return Promise.reject(error);
```

Comment:
Similar handling when we're getting HTTP error code `503`, can both be merged?

Reply:
yeah, I agree with this, we should keep that logic in the same place

Reply:
Fair I moved the first check to be in the same place, but I kept the check from the `/status` endpoint separated so we have better visibility between suspended vs renamed.